frontend/src/app/api/auth/google/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextRequest, NextResponse } from "next/server";
+import { setTokenCookie } from "@/lib/authApi";
+
+export async function POST(req: NextRequest) {
+  const { state } = await req.json();
+
+  const response = await fetch(
+    `http://${process.env.BACKEND_HOST}:${process.env.BACKEND_PORT}/auth/google/login`,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${state}`,
+      },
+    },
+  );
+
+  if (response.ok) {
+    const { accessToken } = await response.json();
+    setTokenCookie(accessToken);
+    return NextResponse.json({ message: "Login successful" });
+  } else {
+    const error = await response.json();
+    return NextResponse.json(error, { status: response.status });
```

Comment:
Argument type {status: number} is not assignable to parameter type ResponseInit | undefined

Reply:
When you received this kind of error?

Reply:
In webstorm IDE (although the code compiles)

Reply:
Could you please provide more details about this error? Specifically, when did you encounter it? Which endpoint were you trying to access, what data did you send, and was Docker running at the time?